src/Storage.js

```diff
@@ -11,13 +11,17 @@ const INSERT_STATEMENT = 'INSERT INTO stream_data '
     + '(id, partition, ts, sequence_no, publisher_id, msg_chain_id, payload) '
     + 'VALUES (?, ?, ?, ?, ?, ?, ?)'
 
+const INSERT_OR_UPDATE_LAST_MESSAHGE = 'UPDATE stream_last_msg SET time = toTimestamp(now()) WHERE id = ?'
+
 const INSERT_STATEMENT_WITH_TTL = 'INSERT INTO stream_data '
     + '(id, partition, ts, sequence_no, publisher_id, msg_chain_id, payload) '
     + 'VALUES (?, ?, ?, ?, ?, ?, ?) USING TTL 259200' // 3 days
 
 const batchingStore = (cassandraClient, insertStatement) => new MicroBatchingStrategy({
     insertFn: (streamMessages) => {
+        const streamIds = new Set()
```

Comment:
No need for `Set` actually since within a batch all messages belong to the same streamId-streamPartition pair.

Reply:
ofc, I can take just first row, get streamId and build `insert`. but what if will change something in `batch logic`? That's why I added this `Set` here.

Reply:
That's a fair point. Can we add a comment to the code to clarify this? Otherwise a reader may be led to believe that in some circumstances the streamIds could be different.
```js
// future-proofing: currently all messages in batch have same streamId, but could change later on  
```